### PR TITLE
download tarball

### PR DESCRIPTION
**Pull request by kristelteng**

### CODE REVIEW

File: src/docs.js [hunk 1]
```diff
@@ -135,8 +135,60 @@ async function documenter(options) {
...
+  options = _.defaults({}, options, {
+    credentials: {},
+    project: null,
+  });
+
+  let auth = new client.Auth({
+    credentials: options.credentials,
+  });
+
+  let creds = await auth.awsS3Credentials('read-write', 'taskcluster-raw-docs', options.project + '/');
```
Comment (imbstack) [inline src/docs.js:151]: I suppose these only need to be read-only, but it's probably not super important to get that fixed at this very moment. We'd create a second set of creds for this that are read-only and pass them in for this part.

**Comment by imbstack** [comment]
Just curious what the interface for this to be used is from tc-docs? Is the returned thing an object mapping filename to file contents?

**Comment by kristelteng** [comment]
@imbstack: Is the returned thing an object mapping filename to file contents? --> yes, it is

File: src/docs.js (hunk 1 above)
Comment (kristelteng) [inline src/docs.js:151]: @imbstack : I can change it to read only

**Pull request merged**